code/modules/trade/datums/trade_stations_presets/zarya.dm

```diff
@@ -39,6 +39,7 @@
 			/obj/item/weapon/tool/omnitool,
 			/obj/structure/reagent_dispensers/fueltank,
 			/obj/machinery/floodlight,
+			/obj/item/weapon/storage/deferred/disks,
```

Comment:
```suggestion
			/obj/item/weapon/storage/deferred/disks
```

Reply:
Why

Reply:
for reference, the last item in a list does not use a comma because the presence of a comma indicates that the list should continue

Reply:
> for reference, the last item in a list does not use a comma because the presence of a comma indicates that the list should continue

If it was c++ or other it would indicate, but it's DM, and it's allow this. And this won't create extra null element if it is last comma in list and this comma haven't anything after it.
![Screenshot_20210529-173342_Discord.jpg](https://user-images.githubusercontent.com/46600554/120067118-1f771700-c0a4-11eb-91d7-681e94ad21b4.jpg)
Better to leave commas on last elements because we have down streams and this commas removing many problems with merge conflicts if someone on downstream will decide to add something to list